demo-projects/meetup/site/components/Talks.js

```diff
@@ -13,19 +13,23 @@ const Talks = ({ talks }) => {
 
         return (
           <Talk key={talk.id}>
-            {hasSpeakers && (
-              <AvatarStack
-                users={talk.speakers}
-                size="large"
-                css={{
-                  marginRight: CONTENT_GUTTER,
-                }}
-              />
-            )}
-            <Content>
+            <div css={{ display: 'flex', alignItems: 'center' }}>
+              {hasSpeakers && (
+                <AvatarStack
+                  users={talk.speakers}
+                  size="large"
+                  css={{
+                    marginRight: CONTENT_GUTTER,
+                  }}
+                />
+              )}
               {talk.isLightningTalk && <Bolt />}
-              <H5 as="h3">{talk.name}</H5>
-              {talks.description ? <Html markup={talk.description} /> : null}
+              <H5 as="h3" css={{ marginLeft: talk.speakers.length ? '1rem' : '0' }}>
```

Comment:
Resolve the DeepScan error:

```suggestion
              <H5 as="h3" css={{ marginLeft: hasSpeakers ? '1rem' : '0' }}>
```